### PR TITLE
Minor grammatical improvements

### PR DESCRIPTION
**Pull request by 3kh0**
## What does this PR do?
Improve repo

## For resources

There are just some minor grammar errors in the README that I fixed up.

### Why is this valuable (or not)?

This makes sure that the README is more readable and understandable.

### How do we know it's really free?

`N/A`

### For book lists, is it a book? For course lists, is it a course? etc.

`N/A`

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction).
- [X] Used an informative name for this pull request.
## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,7 +204,7 @@ Free Podcasts and Screencasts:
 
 ### Programming Playgrounds
 
-Write, compile and run your code within a browser. Try it out!
+Write, compile, and run your code within a browser. Try it out!
```
Comment (marcelbrode) [inline README.md:207]: I would revert that. That makes it worse

Comment (3kh0) [reply README.md]: Ok

**Review by kanak6040 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -37,12 +37,12 @@ The list was moved to GitHub by Victor Felder for collaborative updating and mai
 
 </div>
 
-The [Free Ebook Foundation](https://ebookfoundation.org) now administers the repo, a not-for-profit organization devoted to promoting the creation, distribution, archiving, and sustainability of free ebooks. [Donations](https://ebookfoundation.org/contributions.html) to the Free Ebook Foundation are tax-deductible in the US.
```
Comment (eshellman) [inline README.md:40]: We're not-for-profit. It's a legal term.

File: README.md [hunk 3]
```diff
@@ -37,12 +37,12 @@ The list was moved to GitHub by Victor Felder for collaborative updating and mai
...
+The [Free Ebook Foundation](https://ebookfoundation.org) now administers the repo, a non-profit organization devoted to promoting free ebooks' creation, distribution, archiving, and sustainability. [Donations](https://ebookfoundation.org/contributions.html) to the Free Ebook Foundation are tax-deductible in the US.
 
 
 ## How To Contribute
 
-Please read [CONTRIBUTING](docs/CONTRIBUTING.md). If you're new to GitHub, [welcome](docs/HOWTO.md)! Remember to abide by our adapted from ![Contributor Covenant 1.3](https://img.shields.io/badge/Contributor%20Covenant-1.3-4baaaa.svg) [Code of Conduct](docs/CODE_OF_CONDUCT.md) too ([translations](#translations) also available).
+Please read [CONTRIBUTING](docs/CONTRIBUTING.md). If you're new to GitHub, [welcome](docs/HOWTO.md)! Remember to abide by our adapted [Contributor Covenant 1.3](https://img.shields.io/badge/Contributor%20Covenant-1.3-4baaaa.svg) [Code of Conduct](docs/CODE_OF_CONDUCT.md) too ([translations](#translations) also available).
```
Comment (eshellman) [inline README.md:45]:
```suggestion
Please read [CONTRIBUTING](docs/CONTRIBUTING.md). If you're new to GitHub, [welcome](docs/HOWTO.md)! Also remember to abide by our adapted [Contributor Covenant 1.3](https://img.shields.io/badge/Contributor%20Covenant-1.3-4baaaa.svg) [Code of Conduct](docs/CODE_OF_CONDUCT.md). ([translations](#translations) also available).
```

File: README.md (hunk 2 above)
Comment (eshellman) [reply README.md:40]: `ebooks' creation` is really awkward

File: README.md [hunk 4]
```diff
@@ -24,7 +24,7 @@ This page is available as an easy-to-read website. Access it by clicking on [![h
 
 This list was originally a clone of [StackOverflow - List of Freely Available Programming Books](https://web.archive.org/web/20140606191453/http://stackoverflow.com/questions/194812/list-of-freely-available-programming-books/392926) with contributions from Karan Bhangui and George Stocker.
 
-The list was moved to GitHub by Victor Felder for collaborative updating and maintenance. It has grown to become one of [GitHub's most popular repository](https://octoverse.github.com/).
+The list was moved to GitHub by Victor Felder for collaborative updating and maintenance. It has become one of [GitHub's most popular repository](https://octoverse.github.com/).
```
Comment (eshellman) [inline README.md:27]: it's become what it is by growing.

**Review by eshellman — CHANGES_REQUESTED** [review]
some good suggestions, some bad suggestions.

**Comment by 3kh0** [comment]
@eshellman I have made the changes you requested

**Pull request closed without merging**